server/eclipse-project/src/main/resources/us/freeandfair/corla/default.properties

```diff
@@ -0,0 +1,5 @@
+# The default properties file for ColoradoRLA
+port = 8888
+locale = en_US
+keystore = /us/freeandfair/corla/default.jks
+keystore_password = corla2017
```

Comment:
Well, it's the default properties file (that is, the "if you check out the project and build it without configuration, this is what you run" file). In production, there will be a properties file that actually points to a legit keystore (which, of course, cannot be stored in the repo) and has a legit keystore password. And other properties that have not yet been added there, of course, like "database location".